core/src/main/java/tachyon/web/WebInterfaceBrowseLogsServlet.java

```diff
@@ -117,21 +124,23 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     request.setAttribute("currentPath", "");
 
     String baseDir = mTachyonConf.get(Constants.TACHYON_HOME, Constants.DEFAULT_HOME);
-
+    File logsDir = new File(baseDir, "logs");
     String requestFile = request.getParameter("path");
 
     if (requestFile == null || requestFile.isEmpty()) {
       // List all log files in the log/ directory.
-      DirectoryStream<Path> logFiles =
-              Files.newDirectoryStream(Paths.get(baseDir, "/logs"), "*.log");
+
       List<UiFileInfo> fileInfos = new ArrayList<UiFileInfo>();
-      for (Path logFile : logFiles) {
-        BasicFileAttributes attr = Files.readAttributes(logFile, BasicFileAttributes.class);
-        String logFileName = logFile.getFileName().toString();
-        fileInfos.add(new UiFileInfo(new UiFileInfo.LocalFileInfo(logFileName, logFileName, attr)));
+      File[] logFiles = logsDir.listFiles(mLogFileFilter);
+      for (File logFile : logFiles) {
+        String logFileName = logFile.getName();
+        // java 6 does not support getting creation time, so lastModified is reused.
```

Comment:
Could we just put a default value (and replace it with blank or n/a in the jsp), otherwise it might be misleading?